rvs/conf/MI300X/gst_single.conf

```diff
@@ -24,94 +24,196 @@
 # ###############################################################################
 
 actions:
-- name: gst-1000Tflops-8K-fp8_r
+- name: gst-900Tflops-8K-trig-fp8
   device: all
   module: gst
-  log_interval: 5000
+  log_interval: 3000
   ramp_interval: 5000
-  duration: 30000
+  duration: 15000
   copy_matrix: false
-  target_stress: 1000000
+  target_stress: 900000
   matrix_size_a: 8192
   matrix_size_b: 8192
   matrix_size_c: 8192
+  matrix_init: trig
```

Comment:
mention the usage detail at top of the conf file as a comment

Reply:
Done